app/controllers/rentals_controller.rb

```diff
@@ -41,9 +41,11 @@ def processing
 
   # GET /rentals/1/transform
   def transform
-    if @rental.rental_status == 'reserved'
+    if @rental.reserved? && @rental.end_date < DateTime.current
```

Comment:
One thought and Question for the @umts/parking-programmers: Should we offer the ability to flag a no_show after the beginning or the end of the rental?